results.go

```diff
@@ -122,6 +124,34 @@ func (v *vocdoniHandler) finalizeElectionResults(election *api.Election, electio
 			return
 		}
 		log.Infow("final results image built ondemand", "electionID", election.ElectionID.String())
+		hubResults := &communityhub.HubResults{
+			Question:   electiondb.Question,
+			Options:    choices,
+			Date:       election.EndDate.String(),
+			CensusRoot: election.Census.CensusRoot,
+			CensusURI:  election.Census.CensusURL,
+		}
+		hubResults.Tally = make([][]*big.Int, 0)
+		for _, option := range election.Results[0] {
+			opt := new(big.Int).Set(option.MathBigInt())
+			hubResults.Tally[0] = append(hubResults.Tally[0], opt)
+		}
+		// hubResults.Turnout = big.NewInt(int64(electiondb.Turnout))
+		// hubResults.TotalVotingPower = big.NewInt(int64(election.Census.MaxCensusSize))
+		// hubResults.Participants =
+		if err := v.comhub.SetResults(electiondb.Community.ID, election.ElectionID, hubResults); err != nil {
+			log.Errorw(err, "failed to set results on the community hub")
+		}
+		// check results added to the community hub
+		for i := 0; i < 3; i++ {
```

Comment:
What's the reason for this check? I think SetResults() if no error returned, should either be sure it worked fine or run this check internally .

Reply:
It was just to check the tx was mined. But can be avoided.